packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -36,9 +36,10 @@ import styles from './Toolbar.module.scss';
 
 const cx = classNames.bind(styles);
 
+const DIVIDER_WIDTH = 5;
 const QUERY_INPUT_WIDTH = 175;
-// 4 is marginLeft(2px) + marginRight(2px). also we can simulate divider side margins here
-const TOOLBAR_SQUARE_WIDTH = 40 + 4;
+// 40: square width, 2: left margin, 2: right margin
+const TOOLBAR_SQUARE_WIDTH = 40 + 2 + 2;
```

Comment:
```suggestion
const LEFT_MARGIN = 2;
const RIGHT_MARGIN = 2;
const TOOLBAR_SQUARE_WIDTH = 40 + LEFT_MARGIN + RIGHT_MARGIN;
```
so that's self documented